### PR TITLE
Update ronalds1398.json

### PR DESCRIPTION
**Pull request by Ronalds13424**
I have created this PR to add my TXT record for google indexing.

### CODE REVIEW

File: domains/ronalds1398.json [hunk 1]
```diff
@@ -5,6 +5,7 @@
     "discord": "ronaldino1398"
   },
   "record": {
-    "CNAME": "ronalds-website.pages.dev"
+    "CNAME": "ronalds-website.pages.dev",
+    "TXT": "google-site-verification=YcDGuXFRPF2Udyt8BMXCE7RrQ9wFgvPx1hYAEDFfow8"
```
Comment (CuteDog5695) [inline domains/ronalds1398.json:9]: CNAME can't be with other records.

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/ronalds1398.json (hunk 1 above)
Comment (Ronalds13424) [reply domains/ronalds1398.json:9]: what should i do then?
remove txt?

Comment (cupglassDEV) [reply domains/ronalds1398.json:9]: > what should i do then? remove txt?

yes.

Comment (Ronalds13424) [reply domains/ronalds1398.json:9]: so theres no way i can index my site?

**Pull request closed without merging**